content/components/action-menu.mdx

```diff
@@ -424,6 +424,23 @@ Here is an overview of [action list](/components/action-menu) items that are com
 - **Section header**: To add clarification to a section.
 - **Divider**: To create sections for your selections, actions or submenus.
 
+#### Inactive items
+
+<img
+  width="960"
+  alt="Three examples: 1. An action menu that works just fine; 2. An action menu with some items disabled; 3. An action menu with all items disabled"
+  src="https://github.com/primer/design/assets/2313998/8f3f9bfb-2c0c-4036-a83f-faf17e45ffdb"
+/>
+
+If items in a menu have been affected by a system error such as an outage, they may be put into an inactive state with a message in each item explaining why it's disabled.
+
+If an action menu contains menu items that are very closely related (for example: the action menu for adding a file to a repo) and we can detect that none of the menu items are available, then put the button that triggers the action menu into an [inactive state](/components/button#inactive).
```

Comment:
`[inactive state](/components/button#inactive)` will be broken until https://github.com/primer/design/pull/627 merges